modules/accessing-metrics-outside-cluster.adoc

```diff
@@ -8,44 +8,50 @@
 
 Learn how to query Prometheus statistics from the command line when monitoring your own services. You can access monitoring data from outside the cluster with the `thanos-querier` route.
 
+This access only supports using a Bearer Token for authentication.
+
 .Prerequisites
 
-* You deployed your own service, following the _Enabling monitoring for user-defined projects_ procedure.
+* You have deployed your own service, following the "Enabling monitoring for user-defined projects" procedure.
+* You are logged in to an account with the `cluster-monitoring-view` cluster role, which provides permission to access the Thanos Querier API.
+* You are logged in to an account that has permission to get the Thanos Querier API route.
++
+[NOTE]
+====
+If your account does not have permission to get the Thanos Querier route, a cluster administrator can provide the URL for the route.
```

Comment:
```suggestion
If your account does not have permission to get the Thanos Querier API route, a cluster administrator can provide the URL for the route.
```

Reply:
Not sure if suggestion above is correct,  as I am not aware of the differences between the Thanos Querier  route and Thanos Querier API route, but wanted to mention regardless.

Reply:
Thanks @mletalie! Good catch. It's the same thing, but it doesn't hurt to repeat the whole phrase, I think. I'll make the change.